unison-src/transcripts-round-trip/main.output.md

```diff
@@ -875,7 +875,7 @@ broken tvar =
           (cases
             Some _ ->
               "oh boy isn't this a very very very very very very very long string?"
-            None   -> ""))
+            None -> ""))
```

Comment:
Changes in this file are unrelated. This transcript just hasn't been run for a while.